frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts

```diff
@@ -11,7 +11,7 @@ import { GlobalService } from '../../../services/global.service';
 import { ChallengeService } from '../../../services/challenge.service';
 import { EndpointsService } from '../../../services/endpoints.service';
 import { SelectphaseComponent } from '../../utility/selectphase/selectphase.component';
-import { environment } from '../../../../environments/environment.staging';
+import { environment } from '../../../../environments/environment';
```

Comment:
@ShauryaAg why is this change needed?

Reply:
I am extremely sorry about that. I accidentally created a new branch from my previous branch for another PR. Hence, the commits for my last two PR's are reflected in this one too.

Reply:
I will revert the unnecessary changes in the next commit.